.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug Report
+description: Create a report to help us improve
+title: 'Bug: {short description}'
+labels: bug, needs triage
+---
+
+
+Please take the time to fill out this bug report. Thanks!
+
+## Contact Details
+How can we get in touch with you if we need more info?
+<!--
+ex. email@example.com
+-->
+
+## What happened?
+Tell us what you saw
+
+## What did you expect to happen?
+What were you expecting?
+
+## Step(s) to Reproduce
+Tell us how you came across this
+<!--
+ex. I was on X page, and clicked Y thing
+-->
+
+### Were you on mobile when this happened?
+Yes / No
+
+### What browser(s) are you seeing the problem on?
+- Brave
+- Chrome
+- Firefox
+- Microsoft Edge
+- Opera
+- Safari
+- Vivaldi
+- Other
```

Comment:
You could make these checkboxes.

Reply:
I'm not too worried. We can always just reject or ignore bug reports with insufficient detail/structure.